abjad/tools/quantizationtools/QEventSequence.py

```diff
@@ -659,3 +603,69 @@ def from_tempo_scaled_leaves(cls, leaves, tempo=None):
         # convert durations and pitches to QEvents and return
         return cls.from_millisecond_pitch_pairs(
             tuple(zip(durations, pitches)))
+
+    ### PUBLIC PROPERTIES ###
+
+    @property
+    def duration_in_ms(self):
```

Comment:
Added a re-initialization of `sequence` in the docstring state here to prevent failure from moving these properties belows the methods section.